olm/scc.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: zookeeper-scc
```

Comment:
Currently all custom SCCs are the same. Maybe rename to `stackable-platform-scc` and use it where needed?

Reply:
Done.